tests/block_test.go

```diff
@@ -49,6 +49,11 @@ func TestBlockchain(t *testing.T) {
 	// using 4.6 TGas
 	bt.skipLoad(`.*randomStatetest94.json.*`)
 
+	// The tests under Pyspecs are the ones that are published as execution-spect tests.
+	// We run these tests separately, no need to _also_ run them as part of the
+	// reference tests.
+	bt.skipLoad(`^Pyspecs/`)
```

Comment:
Interesting.  
The exec spec tests are in a different location; the reference tests are at `./testdata/BlockchainTests`, and spec tests at `./spec-tests-fixtures/blockchain_tests`. 

But yes, the exec spec tests are also imported in to the reference tests. For state-tests, it's `GeneralStateTests/Pyspecs` (we should skip those!), for blockchain tests, it seems to be `BlockchainTests/GeneralStateTests/Pyspecs`. That is, those are already skipped due to 
```golang
	// General state tests are 'exported' as blockchain tests, but we can run them natively.
	// For speedier CI-runs, the line below can be uncommented, so those are skipped.
	// For now, in hardfork-times (Berlin), we run the tests both as StateTests and
	// as blockchain tests, since the latter also covers things like receipt root
	bt.skipLoad(`^GeneralStateTests/`)
```